### DISCUSSION TITLE
A compatible drag / drop builder available somewhere?

### DISCUSSION THREAD

**Discussion by ashmetry** [tabler/tabler#1943] [General]
Hi - Does anyone know of a compatible drag drop builder available somewhere? My developer wants me to put together mocks of different pages. My developer is mainly a backend guy who isn't excellent at getting things to be pixel perfect even using tabler.

I'm looking for a quick way to piece components ("like pieces of legos") inside layouts together.

Thanks!

**Comment by martyncoup** [comment]
If it’s just mocks you want, why not use Figma?

On 29 Aug 2024, at 18:37, Ash ***@***.***> wrote:

﻿

Hi - Does anyone know of a compatible drag drop builder available somewhere? My developer wants me to put together mocks of different pages. My developer is mainly a backend guy who isn't excellent at getting things to be pixel perfect even using tabler.

I'm looking for a quick way to piece components ("like pieces of legos") inside layouts together.

Thanks!

**Reply by ashmetry** [reply]
Thanks for your input. I could be wrong, but can Figma generate Tabler HTML that I can zip and give to my developer? My understanding is that Figma is mainly for wireframes. I want to provide him with Tabler pixel-perfect designs that he can plug into the app with the React code that interacts with the actual backend.
